### PR TITLE
[EA Forum only] build a separate user profile for the EA Forum

### PR DESCRIPTION
**Pull request by s-cheng**
As the EA Forum's profile has diverged a lot from the default, we've decided to move our user profile to a separate component.

Example with all the fields filled in (more screenshots in comments below):

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1202572231866438) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/collections/users/custom_fields.ts [hunk 1]
```diff
@@ -916,7 +917,7 @@ addFieldsDict(Users, {
       foreignCollectionName: "Sequences",
       foreignTypeName: "sequence",
       foreignFieldName: "userId",
-      filterFn: sequence => !sequence.draft && !sequence.isDeleted
+      filterFn: sequence => !sequence.draft && !sequence.isDeleted && !sequence.hideFromAuthorPage
```
Comment (s-cheng) [inline packages/lesswrong/lib/collections/users/custom_fields.ts:920]: This will require recalculating `sequenceCount` on the user collection

**Comment by s-cheng** [comment]
Looking at your own profile will show you this extra section with your drafts and other private info, and I think it's the only way to create a new sequence? (Admins can also see this section, but for them it's hidden behind a button click.)

full_profile_own

**Comment by s-cheng** [comment]
Mostly works fine on mobile, since it's a single column:

**Comment by s-cheng** [comment]
A new user with no content will just have one section:

Screen Shot 2022-07-07 at 10 58 21 PM

**Review by Xodarap — APPROVED** [review]
:shipit:

**Pull request merged**